consensus/safety-rules/src/persistent_safety_storage.rs

```diff
@@ -92,19 +92,22 @@ impl PersistentSafetyStorage {
     }
 
     pub fn author(&self) -> Result<Author, Error> {
+        let _ = counters::start_timer("get", OWNER_ACCOUNT);
```

Comment:
Change all of these to be _timer instead of _ as they have different semantics. _ drops immediately while _timer will drop at the end of the scope